lib/neo4j/migrations/helpers.rb

```diff
@@ -0,0 +1,50 @@
+module Neo4j
+  module Migrations
+    module Helpers
+      def rename_property(label, old_property, new_property)
+        by_label(label).where(n: {new_property => nil})
+                       .set("n.#{new_property} = n.#{old_property}")
+                       .remove("n.#{old_property}")
+                       .exec
+      end
+
+      def drop_nodes(label)
+        query.match("(n:`#{label}`)")
+             .optional_match('(n)-[r]-()')
+             .delete(:r, :n).exec
+      end
+
+      def add_labels(label, *labels)
+        by_label(label).set("n:#{labels.join(':')}").exec
+      end
+
+      def remove_labels(label, *labels)
+        by_label(label).remove("n:#{labels.join(':')}").exec
+      end
+
+      alias add_label add_labels
+      alias remove_label remove_labels
+
+      def remove_constraint(label, property)
+        execute("DROP CONSTRAINT ON (n:`#{label}`) ASSERT n.#{property} IS UNIQUE")
```

Comment:
Indeed you can. See https://github.com/neo4jrb/neo4j-core/blob/master/lib/neo4j-server/cypher_label.rb and https://github.com/neo4jrb/neo4j-core/blob/master/lib/neo4j/label.rb. The interface might still have some rough edges. I gave it an overhaul a few months ago but it was borne from a very Java perspective that wasn't exactly friendly...